.github/scripts/fxcore-sync-up-version.js

```diff
@@ -25,9 +25,9 @@ else if(templateVersion.includes("rc")){
     templateConfigFile.version = "0.0.0-rc";
     fse.writeFileSync(templateConfig, JSON.stringify(templateConfigFile, null, 4))
 }
-else if(templateVersion.includes("alpha")) {
-    console.log("sync up template in fx-core as 0.0.0-alpha")
-    templateConfigFile.version = "0.0.0-alpha";
+else if(templateVersion.includes("beta")) {
```

Comment:
can we add more instead of override alpha?

Reply:
updated, please help review again, thanks